vector/src/main/java/im/vector/app/features/navigation/DefaultNavigator.kt

```diff
@@ -246,28 +245,6 @@ class DefaultNavigator @Inject constructor(
 
     override fun requestSessionVerification(fragmentActivity: FragmentActivity, otherSessionId: String) {
         val session = sessionHolder.getSafeActiveSession() ?: return
-<<<<<<< HEAD
-
-        val txId = if (session.cryptoService().crossSigningService().isCrossSigningInitialized()) {
-            session.cryptoService().verificationService().requestKeyVerification(
-                    supportedVerificationMethodsProvider.provide(),
-                    session.myUserId,
-                    listOf(otherSessionId)
-            ).transactionId
-        } else {
-            session.cryptoService()
-                    .verificationService()
-                    .beginKeyVerification(VerificationMethod.SAS, session.myUserId, otherSessionId, null)
-        }
-
-        if (context is AppCompatActivity) {
-            VerificationBottomSheet.withArgs(
-                    roomId = null,
-                    otherUserId = session.myUserId,
-                    transactionId = txId
-            ).show(context.supportFragmentManager, VerificationBottomSheet.WAITING_SELF_VERIF_TAG)
-        }
```

Comment:
This was a tchap changes (the tchap comment was missing). Please keep this code:

```
        // Tchap: Check if cross signing is initialized & support verification between v1 and v2
        val txId = if (session.cryptoService().crossSigningService().isCrossSigningInitialized()) {
            session.cryptoService().verificationService().requestKeyVerification(
                    supportedVerificationMethodsProvider.provide(),
                    session.myUserId,
                    listOf(otherSessionId)
            ).transactionId
        } else {
            session.cryptoService()
                    .verificationService()
                    .beginKeyVerification(VerificationMethod.SAS, session.myUserId, otherSessionId, null)
        }
        VerificationBottomSheet.withArgs(
                roomId = null,
                otherUserId = session.myUserId,
                transactionId = txId
        ).show(fragmentActivity.supportFragmentManager, VerificationBottomSheet.WAITING_SELF_VERIF_TAG)
```